### PR TITLE
Add more redirects

### PR DESCRIPTION
**Pull request by camillobruni**
- versioned redirects for API
- default redirect for v8.dev/tool
- default redirect for v8.dev/api

### CODE REVIEW

**Comment by gsathya** [comment]
maybe we should add a navigation toggle called "Tools"

**Comment by camillobruni** [comment]
I'll add a link in the docs section.

**Comment by gsathya** [comment]
what if we make HEAD as the default and the different versions behind `v8.dev/tools/versions`? HEAD is the one I care about 99% of the time, let's make it easier to access that behind `v8.dev/tools`

**Comment by camillobruni** [comment]
v8.dev/tools/system-analyzer already does that, right.
v8.dev/tools => v8.dev/tools/head seems more consistent

File: firebase.json [hunk 1]
```diff
@@ -59,8 +59,18 @@
...
       },
       {
-        "regex": "/tools/(?P<version>v[0-9]+.*)",
-        "destination": "https://v8.github.io/tools/:version",
+        "regex": "/tools/(?P<version_tool>v[0-9]+.*)",
+        "destination": "https://v8.github.io/tools/:version_tool",
+        "type": 301
+      },
+      {
+        "source": "/tools)",
```
Comment (RReverser) [inline firebase.json:67]: @camillobruni Those parentheses don't look right?

Comment (camillobruni) [reply firebase.json:67]: ups.

File: firebase.json [hunk 2]
```diff
@@ -59,8 +59,18 @@
...
+        "destination": "https://v8.github.io/tools/:version_tool",
+        "type": 301
+      },
+      {
+        "source": "/tools",
+        "destination": "https://v8.github.io/tools/head",
+        "type": 301
+      },
+      {
+        "source": "/tools/versions)",
```
Comment (RReverser) [inline firebase.json:72]: @camillobruni Same here.

**Comment by RReverser** [comment]
@camillobruni Please verify if redirects are working correctly once CI finishes deployment.

**Pull request merged**